### DISCUSSION TITLE
Skiovox wont work (when i do the alt+shift+S part)

### DISCUSSION THREAD

**Discussion by BWUyguygd8ud** [3kh0/ext-remover#957] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Idk what to do. 
Im on V114  and cant update further
![Screenshot 2024-03-04 12 27 40 PM](https://github.com/3kh0/ext-remover/assets/144839558/1296f2e4-29d7-4fef-a7e2-a138049e0904)

**Comment by AshtonDavies** [comment, marked as answer]
SKIOVOX may not work on some versions prior to v118. You might as well freeze any extensions you have or just downgrade for even better exploits while you're on v114.

**Reply by BWUyguygd8ud** [reply]
well it used to work so idk why it wont now

**Reply by BWUyguygd8ud** [reply]
also is there any way to use bookmarklets even tho javascript is blocked and all extensions are blocked

**Reply by AshtonDavies** [reply]
> also is there any way to use bookmarklets even tho javascript is blocked and all extensions are blocked
> 

Which particular bookmarklets?

**Reply by BWUyguygd8ud** [reply]
just like any bookmarklet @AshtonDavies

**Reply by CGRmags** [reply]
@BWUyguygd8ud Off topic but... WHY DO YOU HAVE LIGHT MODE ON

**Reply by CGRmags** [reply]
it burns when i see it

**Reply by BWUyguygd8ud** [reply]
idk

**Reply by AshtonDavies** [reply]
> just like any bookmarklet @AshtonDavies 

You may not have many extensions allowed by your administrator in the Chrome Web Store, but if you are able to install ~~Read&Write~~ (I meant Snap&Read) or uBlock origin, then there are ways for you to run bookmarklets with those.

**Reply by BWUyguygd8ud** [reply]
I HAVE READ AND WRITE

**Reply by AshtonDavies** [reply]
> I HAVE READ AND WRITE
> 

Great! Save a bookmarklet as a website link and click on it.

**Reply by BWUyguygd8ud** [reply]
how do i do that T-T

**Reply by AshtonDavies** [reply]
Oops, sorry. I meant Snap&Read. 🤦
You cannot do it with Read&Write.

**Reply by BWUyguygd8ud** [reply]
OOOO I HAVE THAT TOO

**Reply by BWUyguygd8ud** [reply]
how do i save a website in it tho, and also can you also do it with google lense and if you can how @AshtonDavies

**Reply by BWUyguygd8ud** [reply]
@AshtonDavies u there?

**Reply by AshtonDavies** [reply]
> @AshtonDavies u there?

Please be patient. I will look into it later, because I haven't written instructions for it yet.

**Reply by BWUyguygd8ud** [reply]
ok

**Reply by AshtonDavies** [reply]
@BWUyguygd8ud Alright, when you are logged into the Snap&Read extension and it is turned on, show outlines, or in other words, open the sidebar thing, click the plus button, switch to the sources tab, click the plus at the bottom right to add a new source, and then switch to the website tab.

There, you can set the bookmarklet name in the website title field, and the URL in another field, and then save it.

When it has been saved, you should see your bookmarklet link in the Snap&Read panel that you can click.

**Reply by BWUyguygd8ud** [reply]
@AshtonDavies it didnt work

**Comment by BWUyguygd8ud** [comment]
![Screenshot 2024-03-05 2 50 50 PM](https://github.com/3kh0/ext-remover/assets/144839558/390ff54d-6128-4181-85dd-3c9e46a18d33)

**Comment by BWUyguygd8ud** [comment]
it just shows that, what should I do? @AshtonDavies

**Reply by AshtonDavies** [reply]
That's just a conflict with the quotes. The extension thinks you're ending the URL right there. Switch to single quotes ( ' ) and I think it should work.

**Reply by BWUyguygd8ud** [reply]
it still gives me an error, any other ideas on how to fix

**Reply by AshtonDavies** [reply]
> it still gives me an error, any other ideas on how to fix

Been a while, forgot another part.

After that, go to your outline tab, type anything in the field, then click the blue circle, then click the link icon. It should take you to a screen containing whatever sources, or bookmarklets as we are abusing this feature, lol. There should be checkboxes to the left of each source you have made, but on my side at least, they are invisible and glitched, so just go ahead and click the invisible checkbox if so and then return to your outline screen where you can then click the bookmarklet and run it.

Sorry for all the confusion I gave you, that's my bad.

**Comment by BWUyguygd8ud** [comment]
well now when i click it nothing happens

**Reply by AshtonDavies** [reply]
> well now when i click it nothing happens

What page are you running it on?

**Reply by BWUyguygd8ud** [reply]
edpuzzle

**Reply by BWUyguygd8ud** [reply]
and blooket

**Reply by AshtonDavies** [reply]
Can you run stuff on google.com?

**Reply by BWUyguygd8ud** [reply]
nope

**Reply by AshtonDavies** [reply]
Screenshot your panel.

**Reply by BWUyguygd8ud** [reply]
ima send you a screen recording and you tell me what im doing wrong

**Reply by AshtonDavies** [reply]
> ima send you a screen recording and you tell me what im doing wrong

Okay, that's a good idea.

**Comment by BWUyguygd8ud** [comment]
[Uploading Screen recording 2024-03-05 8.12.04 PM.webm…]()

**Comment by BWUyguygd8ud** [comment]
idk if the video is sending

**Reply by AshtonDavies** [reply]
It's not for some reason. Can you DM me in Discord?

**Reply by BWUyguygd8ud** [reply]
ye

**Reply by BWUyguygd8ud** [reply]
@AshtonDavies i added u
